docs/development/getting-started.rst

```diff
@@ -155,6 +176,21 @@ anything private. The password for every account has been set to the string
 ``password``.
 
 
+Running your developer environment after initial setup
+------------------------------------------------------
+
+You won't have to initialize the database after the first time you do
+so, and you will rarely have to re-run `make build`. Ordinarily, to
+access your developer environment, you'll:
+
+.. code-block:: console
+
+    $ cd warehouse
```

Comment:
This might be confusing because there's another `warehouse` directory within the top-level `warehouse` directory.

Let's assume that all of the commands in this guide are run from the top level of the repo, so the dev will never have to `cd warehouse`.

Reply:
Good point! Will revise. Thanks.